### DISCUSSION TITLE
新题库疑似加入排列组合数学题

### DISCUSSION THREAD

**Discussion by Core-65536** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#19] [General]
![a9b6f581da60de10b7b801e3653fcdd](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/73655341/ae71ee56-eecd-457d-951c-0bdd3d3a1624)
如图，虽然不难，但谁知道以后要搞什么进去（

**Comment by Core-65536** [comment]
这边建议以后可以加点ACM区域银牌水平的算法题😋

**Comment by Core-65536** [comment]
注：消息来源：https://www.oschina.net/news/271547

**Reply by Core-65536** [reply]
这道题如果我高中知识没还给老师的话应该是选B吧？
Ans=(A(7,7)/(A(3,3)*2))*C(8,3)=7840

**Reply by flymeqy** [reply]
正确
